tests/accidental_tests.ts

```diff
@@ -410,10 +412,10 @@ function cautionary(options: TestOptions): void {
 
   const accids = Object.values(accidentals).filter((accid) => accid !== '{' && accid !== '}');
   for (i = 0; i < staveCount; ++i) {
-    const stave = f.Stave({ x: 0, y: 10 + staveHeight / scale * i, width: staveWidth / scale });
+    const stave = f.Stave({ x: 0, y: 10 + (staveHeight / scale) * i, width: staveWidth / scale });
     const score = f.EasyScore();
     const rowMap = [];
-    for (j = 0; j < notesPerStave && (j + i * notesPerStave) < accids.length; ++j) {
+    for (j = 0; j < notesPerStave && j + i * notesPerStave < accids.length; ++j) {
```

Comment:
Also, it looks like the weird parentheses changes are caused by eslint.

Reply:
The first one tries to avoid confusion with a / (b*c)
The second is that eslint considers parenthesis for the whole expression unnecessary.